src/modules/interfaces.ts

```diff
@@ -504,8 +504,8 @@ export class RpcBase implements RpcInterface {
 }
 
 export class AwsSdkInvoker extends RpcBase {
-  preTransactionCheck = PreTransactionCheck.WAIT_FOR_LOCK;
-  postTransactionCheck = PostTransactionCheck.UNLOCK_ALWAYS;
+  preTransactionCheck = PreTransactionCheck.NO_CHECK;
+  postTransactionCheck = PostTransactionCheck.NO_CHECK;
```

Comment:
Should this change still be here?

Reply:
No. Per our discussion on Wednesday's standup meeting this will be rolled back.